src/stdlib_logger.f90

```diff
@@ -569,7 +569,9 @@ end subroutine configure
 
 
     subroutine final_logger( self )
-!! finalizes the logger_t entity by flushing the units
+!! version: experimental
+
+!! finalizes the logger_t entity sekf by flushing the units
```

Comment:
```suggestion
!! finalizes the LOGGER_T entity SELF by flushing the units
```

Reply:
@jvdp1 I was following the general style of the file itself, since they were following a consistent pattern (lower case variables and upper case on docs). I think @wclodius2 already fixed the issue, but I will gladly modify the suggestion.

Reply:
Thank you. Since it is a small typo, I'll commit your suggestion.